test/mapcss/handlersSpec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const chai = require('chai');
+const expect = chai.expect;
+const server = require('../server');
+const mergeDefaults = require('../mergeDefaults');
+const post = require('../../routes/mapcss').post;
+const fs = require('fs');
+const validMapCss = ['way[amenity=clinic][!healthcare]:closed{'
+    + 'throwError: "Health Clinic preset must include healthcare";}'];
+
+const invalidMapCss = ['way [amenity=clinic][!healthcare]:closed{'
+    + 'throwError: "Health Clinic preset must include healthcare";}'];
+
+
+module.exports = () => {
+    before(async () => await server.liftOff(post));
+    describe('post', () => {
```

Comment:
I tried these out and turns out they are now returning false positives.
While they pass, the handler code is never actually being executed.
[Turns out hapi defaults content type of the payload to JSON](https://hapijs.com/api#-await-serverinjectoptions).
 This, for reasons I'm not sure 100% sure, is preventing the request from being handled properly.

Anyway adding the below property at the beginning of both objects you pass into merge defaults will do the trick.

```headers: { 'Content-Type': 'text/plain'}```

I tried this out myself and it revealed that the second test's mapcss can actually be parsed just fine...
Do we want to just add it to the `validMapCss` array for now? 

Now, what to do about testing an invalid post... maybe we can find a honking mapcss selector from the JOSM world to prove that for now, the parser can't handle it :smile: 

Last thing. Even though We've sort of decided that doing the whole schema validation is a bit much, we add an expect statement just as a sanity is what we actually hope it to be.

So something like the following right under the `expect(statusCode).to.equal(200)...

```Javascript
const expected = [{"geometry":"closedway","equals":{"amenity":"clinic"},"absence":"healthcare","error":"{ Health Clinic preset must include healthcare}"}]
expect(JSON.parse(r.payload)).to.equal(expected)
```